app/src/components/CustomPeriodDialog.tsx

```diff
@@ -25,16 +25,20 @@ export function CustomPeriodDialog(props: CustomPeriodDialogProps) {
   const [endTime, setEndTime] = useState<TimeValue>(new Time(12, 0));
 
   const [startDate, setStartDate] = useState<DateValue>(
-    toCalendarDate(new Date(Date.now() - PeriodProps[Period.WEEK].milliseconds))
+      toCalendarDate(new Date(
```

Comment:
This really isn't needed as "a week ago" will always be higher than 2013-01-01

Reply:
Just need a quick clarification - prod presently has 
```typescript
const [startDate, setStartDate] = useState<DateValue>(
    toCalendarDate(new Date(Date.now() - PeriodProps[Period.WEEK].milliseconds))
  );

``` 
Which in-turn has the Start Date set to:
`2023-12-30`
_____________________________________________________
In Dev:
```typescript
const [startDate, setStartDate] = useState<DateValue>(
      toCalendarDate(new Date(
          Math.max(new Date(minDate).getTime())))
  );
``` 
Start Date is:
`2012-02-22`

Is this behavioural change OK?

Reply:
No, startDate's declaration does not need to change

Reply:
The value you put inside of `useState` is the default value, so `startDate` will have a default value of "a week ago", and that's fine, because "a week ago" will never be older than 2013

Reply:
See Commit [178cd22](https://github.com/wise-old-man/wise-old-man/pull/1395/commits/178cd22ef27589b54bd0c02e79efe4773237a5d0). The behaviour you've outlined now matches. Apologies for the drawn out PR.